src/runner.ts

```diff
@@ -36,17 +39,24 @@ export const runner = async () => {
     }
 
     log('Creating shot folders');
+    const createShotsStart = process.hrtime();
     createShotsFolders();
 
     log('Creating shots');
     const shotItems = await createShots();
 
+    const createShotsStop = process.hrtime(createShotsStart);
+    log(`Creating shots took ${parseHrtimeToSeconds(createShotsStop)} seconds`);
+
     if (config.generateOnly && shotItems.length === 0) {
       log(`Exiting process with nothing to compare.`);
+      log(`Process took ${parseHrtimeToSeconds(createShotsStop)} seconds`);
```

Comment:
How about having a `shutdown` function that will include the log message about how long the process took and send the telemetry data. This way we would have a consistent behaviour on every `process.exit()`

Reply:
👌 sounds good - will do this